server/src/util/fs.ts

```diff
@@ -11,3 +12,11 @@ export function getStats(path: string): Promise<Fs.Stats> {
     })
   })
 }
+
+// from https://github.com/sindresorhus/untildify/blob/master/index.js#L11
```

Comment:
```suggestion
// from https://github.com/sindresorhus/untildify/blob/f85a087418aeaa2beb56fe2684fe3b64fc8c588d/index.js#L11
```
Should use the canonical form instead because `master` branch is always changing.